tests/idris2/interface016/expected

```diff
@@ -11,4 +11,4 @@ TwoNum.idr:4:7--4:8
 
 Possible correct results:
     conArg (implicitly bound at TwoNum.idr:4:3--4:8)
-    conArg (implicitly bound at TwoNum.idr:2:1--4:8)
+    conArg (implicitly bound at TwoNum.idr:2:1--5:1)
```

Comment:
Line 5 does not exist so it's a bit strange to see it mentioned.

Reply:
I suspect this is because of how block bounds are parsed at end of input. Let me see what's going  on